### PR TITLE
PegasusAstro SaddlePowerBox

### PR DESCRIPTION
**Pull request by chrysikos**

### CODE REVIEW

File: drivers.xml [hunk 1]
```diff
@@ -645,6 +645,10 @@
             <driver name="Pegasus FlatMaster">indi_pegasus_flatmaster</driver>
             <version>1.2</version>
         </device>
+        <device label="Pegasus SPB" manufacturer="Pegasus Astro">
```
Comment (knro) [inline drivers.xml:648]: I wonder if using a different alias might make this easier? how about "Pegasus Saddle" ?

Comment (chrysikos) [reply drivers.xml:648]: @evansgl is that ok ? Or do you preferer something else?

Comment (evansgl) [reply drivers.xml:648]: Let's use the exact product name:  "Saddle Powerbox"

Comment (knro) [reply drivers.xml:648]: Sounds good. So full name should be "Pegasus Saddle Powerbox" @chrysikos

**Pull request merged**